parser/src/main/kotlin/com/linkedin/dex/spec/EncodedValue.kt

```diff
@@ -126,3 +126,5 @@ sealed class EncodedValue {
         }
     }
 }
+
+private fun Int.toUnsigned8BitInt(): Int = (this and 0xFF)
```

Comment:
Let's put this on `Byte` instead of `Int` to make the call site slightly clearer